src/main/java/dwbh/api/fetchers/UserGroupFetcher.java

```diff
@@ -83,4 +85,19 @@ public boolean isCurrentUserAdmin(DataFetchingEnvironment env) {
     Group group = env.getSource();
     return service.isAdmin(user, group);
   }
+
+  /**
+   * Fetches the users that belongs to a group
+   *
+   * @param env GraphQL execution environment
+   * @return a list of available {@link User}
+   * @since 0.1.0
+   */
+  public List<User> listUsersGroup(DataFetchingEnvironment env) {
+    Context ctx = env.getContext();
```

Comment:
The idea behind fetcher utils is to reuse certain calls like getting the user, if many calls need the user you can create in your fetcher util something like `UserGroupFetcher#getUser`:

```java
static User getUser(DataFetchingEnvironment env) {
   Context ctx = env.getContext();
   return ctx.getAuthenticatedUser();
}
```

and then in your `UserGroupFetcherUtils#listUsersGroup`:

```java
static UserGroupInput listUsersGroup(DataFetchingEnvironment env) {
  User user = getUser(env);
  Group group = env.getSource();

  return new UserGroupInput(user, group);
}
```